backuptar/strconv.go

```diff
@@ -0,0 +1,68 @@
+package backuptar
+
+import (
+	"archive/tar"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Functions copied from https://github.com/golang/go/blob/master/src/archive/tar/strconv.go
```

Comment:
I am not sure if I understand the reason of copying these functions from `strconv.go`. Can you give more details?

Reply:
We need a way to convert to and from PAX-format timestamps for the `LIBARCHIVE.creationtime` field, and archive/tar/strconv.go has perfectly-good functions for this that aren't exported. So I just copied them over, rather than trying to reinvent them from scratch. As noted, this is also what the containerd implementation of this code did.